pkg/helpers/struct.go

```diff
@@ -0,0 +1,309 @@
+// Protocol Buffers - Google's data interchange format
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+// Code generated by protoc-gen-go. DO NOT EDIT.
```

Comment:
I don't think this is true for us here, should we remove this or just note where this code came from?